object_detection/README.md

```diff
@@ -35,7 +35,7 @@ Before You Start:
 * <a href='g3doc/installation.md'>Installation</a><br>
 
 Quick Start:
-* <a href='object_detection_tutorial.ipynb'>
```

Comment:
Please revert. Our intention was to make this viewable on Github (even if it can't be run without compiling protos).